### DISCUSSION TITLE
Readings arrive late The x-drip raises us the reading of the podium for three days

### DISCUSSION THREAD

**Discussion by maram403** [NightscoutFoundation/xDrip#2462] [Q&A]
Readings arrive late
The x-drip raises us the reading of the podium for three days
![image](https://user-images.githubusercontent.com/55768890/201684709-2a76c397-1da9-406f-ac97-0d51998ed8a5.png)

**Comment by maram403** [comment]
> Readings arrive late
> The x-drip raises us the reading of the podium for three days
> ![image](https://user-images.githubusercontent.com/55768890/201684709-2a76c397-1da9-406f-ac97-0d51998ed8a5.png)

**Reply by maram403** [reply]
Main xdrip
When I put a link to my nightscott
The readings arrive late, they raise us three days reading
Readings arrive late
Follow-up through the Shuggah iPhone app

**Comment by Navid200** [comment]
xDrip can be used for many different purposes.  Unless you clarify what you want to do with it, what hardware data source, anyone reading this will have to just guess.  That's not going to help you.

**Reply by maram403** [reply]
Main xdrip
When I put a link to my nightscott
The readings arrive late, they raise us three days reading
Readings arrive late
Follow-up through the Shuggah iPhone app

**Reply by Navid200** [reply]
Do you mean you have readings show up on the xDrip main screen; but, they don't show up in Nightscout if you view Nightscout in a web browser?

**Reply by maram403** [reply]
No

**Reply by Navid200** [reply]
The first step in solving a problem is to understand what the problem is.
I am trying to accomplish that.  

But, I still don't understand what the problem is.
I have no idea if you have readings or not.  If you have no readings, it is not surprising that you will have nothing in Nightscout.

**Comment by maram403** [comment]
![63BCB464-7802-40CD-8708-6D620969C85D](https://user-images.githubusercontent.com/55768890/201770379-08ae7ab8-cb2a-42a6-a50e-754edba955bd.jpeg)
